### ISSUE TITLE
Potential Typo: Variable names may be incorrect in CESST.py lines 907-915 潜在笔误：CESST.py 第907-915行变量名可能有误

### ISSUE THREAD

**Issue by DrunkToast** [AlexYangxx/CESST#4]
非常感谢您分享这么优秀的代码！
Thank you for sharing this excellent code!

我在复现过程中发现一个小问题，不知道是不是复制粘贴错误：
I noticed a minor issue while reproducing your work, which may be a copy-paste error:

File: train_code/architecture/CESST.py   Lines: 907-915行。

当前代码：
 Current Code:
'''
        h_R_2_rb = self.channel_learning(h_R_2, h_B_2)
        h_R_2_rg = self.channel_learning(h_R_2, h_G_2)
        h_R_2 = self.conv(torch.cat([h_R_2_rb, h_R_2_rg], dim=1))
        h_G_2_gr = self.channel_learning(h_G_2, h_R_2)
        h_G_2_gb = self.channel_learning(h_G_2, h_B_2)
        h_G_2 = self.conv(torch.cat([h_G_2_gb, h_G_2_gr], dim=1))
        h_B_2_br = self.channel_learning(h_B_2, h_R_2)
        h_B_2_bg = self.channel_learning(h_B_2, h_G_2)
        h_B_2 = self.conv(torch.cat([h_B_2_br, h_B_2_bg], dim=1))
'''

根据代码上下文的逻辑结构及您论文中的描述，这里应该处理的是第一层特征，修正可能是：
Based on the logical structure of the code context and the description in your paper, this should be processing the first-layer features.
Suggested Fix: 
'''
        h_R_1_rb = self.channel_learning(h_R_1, h_B_1)
        h_R_1_rg = self.channel_learning(h_R_1, h_G_1)
        h_R_1 = self.conv(torch.cat([h_R_1_rb, h_R_1_rg], dim=1))
        h_G_1_gr = self.channel_learning(h_G_1, h_R_1)
        h_G_1_gb = self.channel_learning(h_G_1, h_B_1)
        h_G_1 = self.conv(torch.cat([h_G_1_gb, h_G_1_gr], dim=1))
        h_B_1_br = self.channel_learning(h_B_1, h_R_1)
        h_B_1_bg = self.channel_learning(h_B_1, h_G_1)
        h_B_1 = self.conv(torch.cat([h_B_1_br, h_B_1_bg], dim=1))
'''

最后再次感谢您的开源贡献！
Thanks again for your open-source contribution!
